src/components/Login/Login.js

```diff
@@ -11,6 +14,8 @@ import "./Login.scss";
 const Login = () => {
   const [email, setEmail] = useState("");
   const [password, setPassword] = useState("");
+  const [authError, setAuthError] = useState("");
```

Comment:
Here we are initializing `authError` with an empty string, but then we set it to a boolean.
I think what we should do is set it to false an rename `authError` and `setAuthError` to something that describe better what it is, as follows:
```suggestion
    const [hasAuthError, setHasAuthError] = useState(false);
```